modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/rel/IgniteIndexScan.java

```diff
@@ -39,13 +41,18 @@
     /** */
     private final long sourceId;
 
+    /** */
+    private final RelCollation idxCollation;
```

Comment:
let's rename this to just `collation`

Reply:
There is already `collation` (in traits), so it will be confusion. (but `idxCollation` already removed)